src/Compilers/CSharp/Portable/Declarations/DeclarationTreeBuilder.cs

```diff
@@ -898,6 +966,54 @@ static bool checkPropertyOrFieldMemberForRequiredModifier(Syntax.InternalSyntax.
             }
         }
 
+        private ImmutableSegmentedHashSet<string> GetOrComputeMemberNames<TData>(
+            SyntaxNode parent,
+            Action<HashSet<string>, TData> addMemberNames,
+            TData data)
+        {
+            // Try to obtain the prior member names computed for the corresponding type decl (in lexicographic order)
+            // from the prior version of this tree.
+            var previousMemberNames = _currentTypeIndex < _previousMemberNames.Count
```

Comment:
>var previousMemberNames = _currentTypeIndex < _previousMemberNames.Count [](http://example.com/codeflow?start=12&length=72)

Please move this local into the ```if```, it looks like it is not used outside its body.

Reply:
Done :)